bower.json

```diff
@@ -17,13 +17,15 @@
     "tests"
   ],
   "dependencies": {
-    "angular": "^1.5.3",
-    "angular-animate": "^1.5.3",
+    "angular": "1.5.5",
```

Comment:
No reason at all, I was just used from hawkular to use =version. I'll change it back to `"angular": "^1.5.3",`